### PR TITLE
Added rerun.sh prorotype

### PR DESCRIPTION
**Pull request by brngylni**
I have made a script to be a prototype and give an idea what can be done. It is relevant with #2511 
We can discuss the problems of this script and how to solve them under here.   

Signed-off-by: Mehmet Geylani <barankara5@gmail.com>

### CODE REVIEW

File: rerun.sh [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	elif [[ "${j[0]}" == *"TKG.git"* ]] 
+	then
+		echo "Fetching TKG..."
+		cd openjdk-tests	
+		echo "git clone ${j[0]} "
+		git clone "${j[0]}" 
+		cd "$(basename "${j[0]}" .git)" 
+		git checkout "${j[1]}"
+		cd ..
+		cd ..
```
Comment (llxia) [inline rerun.sh:121]: We should call [get.sh](https://github.com/adoptium/aqa-tests/blob/master/get.sh) for git clone openj9 and TKG repo and move folders.

Comment (brngylni) [reply rerun.sh:121]: But don't we need to checkout before moving folders?

Comment (llxia) [reply rerun.sh:121]: The logic for git clone openj9 and TKG repo and moving folders are in get.sh. There is no need to duplicate the logic. We can just call get.sh and passing the required parameters.
https://github.com/adoptium/aqa-tests/blob/master/get.sh#L426
https://github.com/adoptium/aqa-tests/blob/master/get.sh#L465

The only exception is git clone https://github.com/adoptium/aqa-tests repo, which we do need to have.

Comment (brngylni) [reply rerun.sh]: Check my last commit please.

**Review by llxia — APPROVED** [review]
A couple of enhancements are needed., but I am ok to deliver it as a prototype.

**Comment by smlambert** [comment]
@brngylni - can you please switch this from Draft to PR now?  thanks!

**Review by smlambert — APPROVED** [review]
thanks @brngylni !

**Pull request merged**